wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights: Loading View

Refs. #10379

This PR adds a loading view for the Stats Insights section.

![stats-insights-loading-view](https://user-images.githubusercontent.com/912252/57017316-83edf080-6c16-11e9-9288-065bbdc75feb.gif)

## NoResultsViewHost
Because we often duplicate the code to add the NoResultsViewController as child, I thought to create a helper using a protocol called `NoResultsViewHost` (thanks @frosty for the name suggestion). Its extension constrained to a UIViewController contains the logic to configure, display and hide the noResultsViewController object. Also using associated objects allows us to implement the protocol without any other particular implementation.

## Error Handling
I will push another PR for the error handling. This because it requires some change on the Store.

## To test:
• From _My Sites_ select a site and open its dashboard
• Select _Stats_ to open the Stats section. Be sure the tab _Insights_ is selected.
• You should see the loading view
• The loading view should disappear as soon the data is loaded

Update release notes:
